content/docs/hardware/parts/_index.en.md

```diff
@@ -89,16 +90,81 @@ checking everything is important.
 </tr>
+
 <tr>
   <td>Passive and various components</td>
-  <td colspan="3" align="center">
-    Purchase according to <a href="https://www.reichelt.de/my/1746485">this list</a>
-  </td>
+  <td colspan="3" align="center">see notes</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/jst-crimpkontakt-buchse-xh-jst-xh-ckb-p185091.html">JST contact BXH-001T-P0.6</a></td>
+  <td colspan="3" align="center">15</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/jst-crimpkontakt-buchse-xh-jst-xh-ckb-p185091.html">JST housing XHP-2</a></td>
+  <td colspan="3" align="center">3</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/jst-buchsengehaeuse-1x4-polig-xh-jst-xh4p-bu-p185087.html">JST housing XHP-4</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/jst-buchsengehaeuse-1x5-polig-xh-jst-xh5p-bu-p185088.html">JST housing XHP-5</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/jst-stiftleiste-gerade-1x2-polig-xh-jst-xh2p-st-p185073.html">JST shrouded header B2B-XH-A</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/jst-stiftleiste-gerade-1x5-polig-xh-jst-xh5p-st-p185076.html">JST shrouded header B5B-XH-A</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/widerstand-kohleschicht-10-kohm-0207-250-mw-5--1-4w-10k-p1338.html">resistor (CFR) 10 kΩ, 250 mW, 5 %</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/widerstand-kohleschicht-150-kohm-0207-250-mw-5--1-4w-150k-p1355.html">resistor (CFR) 150 kΩ, 250 mW, 5 %</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/widerstand-kohleschicht-300-kohm-0207-250-mw-5--1-4w-300k-p1407.html">resistor (CFR) 300 kΩ, 250 mW, 5 %</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.conrad.de/de/p/tru-components-tc-1206s4f1101t5e203-dickschicht-widerstand-1-1-k-smd-1206-0-25-w-1-1-st-tape-cut-1584104.html">SMD resistor 1.1 kΩ, 1206, 0.25 W, 1 %</a></td>
+  <td colspan="3" align="center">2</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/vielschicht-keramikkondensator-100n-10--x7r-2-5-100n-p22853.html">ceramic capacitor 100 nF, 10 %</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/elko-radial-22-uf-10-v-1000-h-low-esr-aec-q200-rad-fc-22-10-p84587.html">radial electrolytic capacitor 22 µF, 10 V</a></td>
+  <td colspan="3" align="center">2</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/schottkydiode-60-v-5-a-do-201ad-sb-560-p16081.html">Schottky diode, 60 V, 5 A</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/rueckstellende-sicherungen-max-40a-30v-6-6s-pfra-110-p35211.html">resettable fuse, max. 40 A - 30 V, 6.6 sec</a></td>
```

Comment:
I sadly don't know what the package is called. It seems to be "radial" (considering it is mounted vertically, like the caps, not horizontally, like the axial resistors), and we could specify the pitch.

Reply:
The type has been added.